Makefile

```diff
@@ -51,8 +51,10 @@ upgrade-build: #TODO: reenable it
 
 .PHONY: osdk-image
 osdk-image:
+ifndef IMAGE_TAG
+ override IMAGE_TAG = quay.io/rhpipeline/openshift-pipelines-operator:test
```

Comment:
I think you can just use : 

```
IMAGE_TAG := quay.io/rhpipeline/openshift-pipelines-operator:test
```

and having `make IMAGE_TAG=foo` would redefine it,

Reply:
ah, you are absolutely right. 👍